src/Dialog.cpp

```diff
@@ -4079,7 +4081,7 @@ BOOL CStdRaum::PreLButtonDown(CPoint point) {
                 StopDialog();
                 break;
 
-                //Übernamedialog:
+                // Übernamedialog:
```

Comment:
There seems to be some Umlaute broken in comments

Reply:
Looks fine to me. There was one space added by my VS Code auto formatter.

Reply:
Seems to be a bug in the GitHub diff view then, checked the file, looks good there